bin/format-grafana-dashboards.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
```

Comment:
So this is a one time, or infrequent activity to run this ?

Perhaps add a comment on when someone needs to run it